app/controllers/agentAppointment/AppointAgentController.scala

```diff
@@ -255,9 +256,9 @@ class AppointAgentController @Inject()(
                 .map(oar => oar.copy(filterTotal = oar.authorisations.size))
                 .map(oar => oar.copy(authorisations = oar.authorisations.take(pagination.pageSize)))
             } yield {
-              BadRequest(views.html.propertyrepresentation.revokeAgentProperties(Some(errors), AppointAgentPropertiesVM(group, response), PaginationParameters(), GetPropertyLinksParameters(), group.agentCode))
+              BadRequest(views.html.propertyrepresentation.revokeAgentProperties(Some(errors), AppointAgentPropertiesVM(group, response), PaginationParameters(), GetPropertyLinksParameters(), agentCode))
             }
-          case None =>
+          case None | Some(_) =>
```

Comment:
` case None | Some(_) ` ?  if it either is or isn't there?  use `case _` for catch-all